src/help/getOptionsFromElement.js

```diff
@@ -1,25 +1,25 @@
 import optionsFromStrings from "./optionsFromStrings.js";
-import defaults from "../options/defaults.js";
 
 function getOptionsFromElement(element){
```

Comment:
As a suggestion:

```js
function getOptionsFromElement(element) {
	const options = {};
	const attributes = element.attributes;

	for (let i = 0; i < attributes.length; i++) {
		const match = attributes[i].name
			.match(/(jsbarcode|data)-([A-Za-z0-9-]+)/i);

		if (match) {
			// Transforms foo-bar to fooBar
			const property = match[2].replace(/-[a-zA-Z]/, (val) => val[1].toUpperCase());
			options[property] = attributes[i].value;
		}
	}

	// Since all attributes are string they need to be converted to integers
	return optionsFromStrings(options);
}
```